src/xenia/kernel/xboxkrnl_memory.cc

```diff
@@ -306,8 +306,8 @@ SHIM_CALL MmAllocatePhysicalMemoryEx_shim(PPCContext* ppc_state,
   // and the memory must be allocated there. I haven't seen a game do this,
   // and instead they all do min=0 / max=-1 to indicate the system should pick.
   // If we have to suport arbitrary placement things will get nasty.
-  assert_true(min_addr_range == 0);
-  assert_true(max_addr_range == 0xFFFFFFFF);
+  //assert_true(min_addr_range == 0);
```

Comment:
does this seem work as expected? (I think I implemented AllocRange right, but haven't tested it through this path)
if so, let's remove the asserts.